hack/generate/samples/internal/go/memcached_with_webhooks.go

```diff
@@ -59,6 +60,8 @@ func (mh *MemcachedGoWithWebhooks) Prepare() {
 func (mh *MemcachedGoWithWebhooks) Run() {
 	log.Infof("creating the project")
 	err := mh.ctx.Init(
+		// TODO(estroz): remove this once go/v3-alpha is stabilized and the default plugin.
+		"--plugins", "go/v3-alpha",
```

Comment:
I understand that we will not remove the support for go/v2 (default plugin). So, if we will have both plugins we ought to have both samples. See: https://github.com/operator-framework/operator-sdk/pull/4232#issuecomment-727192170

Reply:
Updated the samples generator to create both `testdata/go/v2` and `testdata/go/v3`.